src/batches/batches.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { CreateBatchDto } from './dto/create-batch.dto';
+//import { UpdateBatchDto } from './dto/update-batch.dto';
+import { PrismaService } from './../prisma/prisma.service';
+
+@Injectable()
+export class BatchesService {
+  constructor(private prisma: PrismaService) {}
+
+  create(createBatchDto: CreateBatchDto) {
+    return this.prisma.batch.create({ data: createBatchDto });
```

Comment:
i don't see `batch` defined in `PrismaService`, what's the type, this should be failing the linter.

Reply:
No, it doesn't fail because the `batch` is defined as a property of the `prisma` object and not directly inside the PrismaService class.
 `prisma.batch` Exposes CRUD operations for the Batch model.

Reply:
The linter should be failing.